Comp2024/src/main/java/frc/robot/Constants.java

```diff
@@ -212,27 +212,6 @@ public enum RotaryMode
   /////////////////////////////////////////////////////////////////////////////
   public static final class SHConsts
   {
-    // Global settings
```

Comment:
Almost every one of these constants need to go into the Shooter subsystem as private constants.